akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala

```diff
@@ -78,12 +78,8 @@ object DispatcherDocSpec {
       executor = "thread-pool-executor"
       # Configuration for the thread pool
       thread-pool-executor {
-        # minimum number of threads to cap factor-based core number to
-        core-pool-size-min = 2
-        # No of core threads ... ceil(available processors * factor)
-        core-pool-size-factor = 2.0
-        # maximum number of threads to cap factor-based number to
-        core-pool-size-max = 10
+        // or in Akka 2.4.2+
+        fixed-pool-size = 16
```

Comment:
I don't understand this change. The documentation where this is included is about how to configure a dispatcher https://raw.githubusercontent.com/akka/akka/master/akka-docs/rst/scala/dispatchers.rst

The original example is valid. The `fixed-pool-size` is an alternative way of doing it when you don't care about the factor.

I suggest that we add another example to the docs, below the original one.

```
And here's another example that uses the "thread-pool-executor" with a fixed thread pool size:

.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#fixed-pool-size-dispatcher-config
```

Note that we already have that example (used further down in the page).